internal/util/util.go

```diff
@@ -76,3 +77,14 @@ func ResolveConfigFilePaths(config *config.MountConfig) (err error) {
 	}
 	return
 }
+
+// Returns empty String in case of failure in marshalling the passed input.
```

Comment:
nit: Stringify marshals an object (only exported attribute) to a JSON string. If marshalling fails, it returns an empty string.

Reply:
done